ontopy/excelparser.py

```diff
@@ -99,18 +100,19 @@ def create_ontology_from_excel(  # pylint: disable=too-many-arguments
     """
     # Get imported ontologies from optional "Imports" sheet
     if not imports:
-        imports = []
+        imports = pd.DataFrame()
+
     try:
-        imports_frame = pd.read_excel(
+        imports = pd.read_excel(
             excelpath, sheet_name=imports_sheet_name, skiprows=[1]
         )
     except ValueError:
         pass
     else:
-        # Strip leading and trailing white spaces in path
-        imports.extend(
-            imports_frame["Imported ontologies"].str.strip().to_list()
-        )
+        # Strip leading and trailing white spaces in paths
+        imports = imports.replace(" ", "")
```

Comment:
This removes all spaces in path. Use `imports.strip()` to strip leading and trailing white spaces.

Reply:
Good point. I did not consider the possibility of adding the path to a local file with spaces in the path.